.github/workflows/self-nightly-past-ci-caller.yml

```diff
@@ -2,32 +2,30 @@ name: Self-hosted runner (nightly-past-ci-caller)
 
 on:
   schedule:
-    # 2:17 am on each Sunday and Thursday
-
-    - cron: "17 2 * * 0,4"
+    - cron: "17 2,14 * * *"

```

Comment:
trigger twice per day

Reply:
Why twice a day?

Reply:
The objective is simply to have the runs of all past versions finished in one weak.
We have currently 10 versions to run (and more after I add the torch 2.x versions).

And each run could be done within 12 hours, so that's why I put twice a day.
But that is not a hard call, we can change if we want.